common/src/main/java/com/scottlogic/deg/common/profile/Field.java

```diff
@@ -20,9 +20,11 @@
 
 public class Field {
     public final String name;
+    public boolean unique;
 
-    public Field(String name) {
```

Comment:
i wouldn't want this to be part of the field definition, if it's a constraint.

Reply:
Is the best way to handle this to extract it out to a wrapper? ie. `FieldUniquenessWrapper`, containing a `Field` and a `boolean`?